agent_metrics/README.md

```diff
@@ -30,7 +30,11 @@ The Agent Metrics integration can potentially emit [custom metrics][7], which ma
 [Run the Agent's status subcommand][9] and look for `agent_metrics` under the Checks section.
 
 ## Data Collected
+
+All data collected are only available for Agent v5.
```

Comment:
curiosity: Apparently, there is an interesting debate about this: https://www.theguardian.com/news/datablog/2010/jul/16/data-plural-singular